Maple2.Database/Model/WeddingHall.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Maple2.Database.Extensions;
+using Maple2.Model.Enum;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Maple2.Database.Model;
+
+internal class WeddingHall {
+    public long Id { get; set; }
+    public long MarriageId { get; set; }
```

Comment:
can't these two ids become one?

Reply:
It shouldn't be the same. A couple can change or cancel their reservation with totally different times, locations, etc. When sending wedding invitations out, that invitation is only valid for that specific wedding hall. keeping it the same ID may allow un-invited guests to show up on the changed reservation.